docs/Using-Fleet/REST-API.md

```diff
@@ -5222,17 +5240,25 @@ Returns a list of all queries in the Fleet instance.
-| ---------------- | ------ | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| name             | string | body | **Required**. The name of the query.                                                                                                                   |
-| query            | string | body | **Required**. The query in SQL syntax.                                                                                                                 |
-| description      | string | body | The query's description.                                                                                                                               |
-| observer_can_run | bool   | body | Whether or not users with the `observer` role can run the query. In Fleet 4.0.0, 3 user roles were introduced (`admin`, `maintainer`, and `observer`). This field is only relevant for the `observer` role. The `observer_plus` role can run any query and is not limited by this flag (`observer_plus` role was added in Fleet 4.30.0). |
+| Name                            | Type    | In   | Description                                                                                                                                            |
+| ------------------------------- | ------- | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| name                            | string  | body | **Required**. The name of the query.                                                                                                                   |
+| query                           | string  | body | **Required**. The query in SQL syntax.                                                                                                                 |
+| description                     | string  | body | The query's description.                                                                                                                               |
+| observer_can_run                | bool    | body | Whether or not users with the `observer` role can run the query. In Fleet 4.0.0, 3 user roles were introduced (`admin`, `maintainer`, and `observer`). This field is only relevant for the `observer` role. The `observer_plus` role can run any query and is not limited by this flag (`observer_plus` role was added in Fleet 4.30.0). |
+| team_id                         | integer | body | The parent team to which the new query should be added. If not submitted or blank, the query will be global.                                           |
+| frequency                       | integer | body | The amount of time, in seconds, the query waits before running. Can be set to `0` to never run. Default: 0.       |
```

Comment:
@rachaelshaw I realized that these are different from how the scheduled queries are described below. I left the scheduled queries stuff below alone. I assume we are comfortable with the new system behaving differently (different names and nesting of the automation stuff) from the old scheduled queries, but want to call that out.